main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1835,7 +1835,8 @@ private void loadDescription(final CacheDetailActivity activity, final String de
                     descriptionView.setMovementMethod(AnchorAwareLinkMovementMethod.getInstance());
                     descriptionView.setVisibility(View.VISIBLE);
                     activity.addContextMenu(descriptionView);
-                    activity.potentiallyHideShortDescription();
+                    // TODO fix the non working method, see https://github.com/cgeo/cgeo/issues/11455
+                    //activity.potentiallyHideShortDescription();
```

Comment:
I propose to remove the method `potentiallyHideShortDescription` as well, as the commented-out line is the only place calling this non-functional method.

Reply:
I just disabled the call and left the method as it is to fix the problem later.
For this point I created Issue #11464.

Reply:
I deactivated the called method now.